### PR TITLE
OFS-281: Create GraphQL queries for the Insuree policy

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OFS-281

requires changes from that PR in Insuree module

### CODE REVIEW

File: contract/signals.py [hunk 1]
```diff
@@ -88,9 +91,36 @@ def append_contract_filter(sender, **kwargs):
...
+        # then check perms
+        if user.has_perms(InsureeConfig.gql_query_insuree_policy_perms) or user.has_perms(PolicyholderConfig.gql_query_insuree_policy_portal_perms):
+            contract_id = additional_filter["contract"]
+            contract_to_process = Contract.objects.filter(id=contract_id).first()
+            # check if user is linked to ph in policy holder user table
+            type_user = f"{user}"
+            # related to user object output (i) or (t)
+            # check if we have interactive user from current context
+            if '(i)' in type_user:
+                ph_user = PolicyHolderUser.objects.filter(Q(policy_holder__id=contract_to_process.policy_holder.id, user__id=user.i_user.id)).first()
```
Comment (delcroip) [inline contract/signals.py:108]: I just realized that you are not making sure the access is still valid should add something like
validFrom <= now(), Q(validTo__isnull=True )|Q(validTo>now() )

Comment (delcroip) [reply contract/signals.py:108]: that is true for all signals

**Review by delcroip — CHANGES_REQUESTED** [review]

File: contract/signals.py (hunk 1 above)
Comment (sniedzielski) [reply contract/signals.py:108]: Ok. I will add this to check this.

Comment (sniedzielski) [reply contract/signals.py]: @delcroip I added that condition to check validity of that PHUser. Later in OFS-284 I will add (probably tomorrow) for another signals in PH and Contract for Payment  query processing.

**Comment by sniedzielski** [comment]
@delcroip You can re-check this PR after my fix.

**Review by delcroip — APPROVED** [review]

**Pull request merged**